tests/check_odov2.sh

```diff
@@ -204,9 +204,9 @@ for stack in $STACKS; do
   devfile_name=$($YQ_PATH eval '.metadata.name' $devfile_path)
   devfile_version=$($YQ_PATH eval '.metadata.version' $devfile_path)
 
-  # Skipping the java-wildfly-bootable-jar stack right now since it's broken.
+  # Skipping the java-wildfly-bootable-jar and java-quarkus stacks right now since they are broken.
   # TODO: Uncomment once fixed.
-  if [[ $stack != "java-wildfly-bootable-jar" && $stack != "java-quarkus/1.3.0" && $stack != "java-quarkus/1.4.0" ]]; then
+  if [[ $stack != "java-wildfly-bootable-jar" && $stack != "java-quarkus" ]]; then
```

Comment:
When running the check script it still includes `java-quarkus`, I believe it is because `$stack` includes the version number as well (e.g. `java-quarkus/1.4.0`) so any stack with multiple versions in this condition is not being skipped.

To fix this I'd suggest using the `$devfile_name` itstead:
```suggestion
  if [[ $devfile_name != "java-wildfly-bootable-jar" && $devfile_name != "java-quarkus" ]]; then
```